index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/Styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+  </head>
+  <body>
+    <section class="navigation">
+      <h3>Byiringiro Moise</h3>
+      <span class="material-symbols-outlined"> menu </span>
```

Comment:
- [ ] I think it will be better if you use `<header>` tag instead of `<section class="navigation">` in your toolbar to make your website semantic because the header represents the introductory section of your website which will thus increase your website accessibility 😄 . You can check out this [link](https://github.com/XRichardroidX/My-Microverse-Portfolio/pull/div%20class=%22head%22) for more information 👍🏽